bridges/testing/environments/rococo-westend/bridges_rococo_westend.sh

```diff
@@ -270,7 +279,7 @@ case "$1" in
           "//Alice" \
           1000 \
           "ws://127.0.0.1:9910" \
-          "$(jq --null-input '{ "parents": 2, "interior": { "X1": [{ "GlobalConsensus": "Westend" }] } }')" \
+          "$(jq --null-input --argjson genesis $WESTEND_GENESIS_HASH '{ "parents": 2, "interior": { "X1": [{ "GlobalConsensus": { ByGenesis: $genesis } }] } }')" \
```

Comment:
Nit: Do we need the `-argjson genesis $WESTEND_GENESIS_HASH` ? Can't we just provide the `$WESTEND_GENESIS_HASH` directly to the `ByGenesis: ...` ?

Reply:
Ok, fixed here: https://github.com/paritytech/polkadot-sdk/pull/6465/commits/469b774e4dd256c2ca9a6c1c34db5e2fb6c2df2e